agent/ecscni/netconfig_windows.go

```diff
@@ -0,0 +1,89 @@
+// +build windows
```

Comment:
can you add some unit tests for this file? existing netconfig.go doesn't have unit tests but i guess that wouldn't be a good reason for not having tests for more code